com.unity.netcode.gameobjects/Editor/NetworkManagerHelper.cs

```diff
@@ -65,7 +65,11 @@ private static void ScenesInBuildActiveSceneCheck()
             var scenesList = EditorBuildSettings.scenes.ToList();
             var activeScene = SceneManager.GetActiveScene();
             var isSceneInBuildSettings = scenesList.Count((c) => c.path == activeScene.path) == 1;
+#if USE_FINDOBJECTSBYTYPE
```

Comment:
use `#if UNITY_2023_1_OR_NEWER` macro (or a similar one) instead of our very own asmdef ones?

Reply:
It is specific to 2023.1.0a21 or newer and I couldn't find a define that already existed for just that version and up. 
Will look around again to see if there is already a define specific to that update.

Reply:
usually, `UNITY_X_Y_OR_NEWER` macros show themselves in release (and beta) versions. if 2023 specific ones are not there, maybe this PR is "too early".

Reply:
Just pushed the pragma approach when UNITY_2023_1_OR_NEWER is defined.